client/src/modules/inventory/list/list.html

```diff
@@ -35,7 +35,7 @@
 
             <li role="separator" class="divider"></li>
             <li role="menuitem">
-              <a ng-href="/reports/inventory/items/?params={{ InventoryCtrl.parameters }}&lang={{ InventoryCtrl.lang }}"
+              <a ng-href="/reports/inventory/items/?params={{ InventoryCtrl.filters.formatHTTP(true) }}&lang={{ InventoryCtrl.lang }}"
```

Comment:
Nice!

Reply:
Unfortunately, this doesn't fix the inventory list :(.  I've made an issue to be tackled separately https://github.com/IMA-WorldHealth/bhima-2.X/issues/2232.